helios-testing/src/main/java/com/spotify/helios/testing/HeliosSoloDeployment.java

```diff
@@ -176,18 +203,12 @@ private boolean isBoot2Docker(final Info dockerInfo) {
     return dockerInfo.operatingSystem().contains(BOOT2DOCKER_SIGNATURE);
   }
 
-  private Info dockerInfo() {
-    try {
-      return this.dockerClient.info();
-    } catch (DockerException | InterruptedException e) {
-      // There's not a lot we can do if Docker is unreachable.
-      throw Throwables.propagate(e);
-    }
+  private boolean isDockerForMac(final Info dockerInfo) {
+    return "moby".equals(dockerInfo.name());
```

Comment:
potentially not but it is the best identifier I found for now. As I mentioned in the PR summary, I feel like there should be a broader condition to test for that I am not able to see the answer to ("what IP to use to reach a mapped port?") but I wanted to at least improve things for these users.